### PR TITLE
[enhancement] clear useless txn log

### PR DESCRIPTION
**Pull request by abc982627271**
## What type of PR is this：
- [ ] bugfix
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 30 / 34 (88.24%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/alter/SchemaChangeHandler.java|8|10|80.00%|[1045, 1048]|
|:large_blue_circle:|com/starrocks/system/HeartbeatMgr.java|10|12|83.33%|[91, 95]|
|:large_blue_circle:|com/starrocks/transaction/DatabaseTransactionMgr.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/transaction/GlobalTransactionMgr.java|7|7|100.00%|[]|

**Review by sduzh — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java [hunk 1]
```diff
@@ -263,6 +280,9 @@ public HeartbeatResponse call() {
                 long flags = heartbeatFlags.getHeartbeatFlags();
                 copiedMasterInfo.setHeartbeat_flags(flags);
                 copiedMasterInfo.setBackend_id(computeNodeId);
+                copiedMasterInfo.setMin_active_txn_id(computeMinActiveTxnId());
+                // for debug
+                LOG.info("min active txn id in HeartBeatMgr is {}", computeMinActiveTxnId());
```
Comment (sduzh) [inline fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java:285]:
```suggestion
```

**Review by sduzh — DISMISSED** [review]

**Review by sduzh — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by sduzh** [comment]
run starrocks_admit_test

**Pull request merged**